src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Markup/Primitives/ExtensionSimplifierMarkupObject.cs

```diff
@@ -355,16 +355,16 @@ private string ConvertMarkupItemToString(MarkupObject item)
                     Debug.Assert(!propertyWritten, "An argument was returned after a property was set. All arguments must be returned first and in order");
                 }
                 
-                string value = property.StringValue;
+                ReadOnlySpan<char> value = property.StringValue;
                 
-                if (value != null && value.Length > 0)
+                if (value.Length > 0)
```

Comment:
In some other files, you used `span.IsEmpty` by there you used `value.Length > 0`. I think we should choose one or the other (I personally prefer IsEmpty, if there are no perf implications).

Reply:
I'd just kept the original code here, but I don't mind changing it to IsEmpty.